viam-cartographer/src/carto_facade/carto_facade_test.cc

```diff
@@ -630,7 +649,7 @@ BOOST_AUTO_TEST_CASE(CartoFacade_demo) {
         VLOG(1) << "viam_carto_add_sensor_reading 1";
         viam_carto_sensor_reading sr = new_test_sensor_reading(
             "sensor_1", ".artifact/data/viam-cartographer/mock_lidar/0.pcd",
-            1687900053773475);
+            1629037851000000);
```

Comment:
Change to be only 1 second between readings.

Reply:
Also add test for get internal state after the first reading

Reply:
Are you saying that this code change is due to us now saying there's 1 second between readings?